analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SyntaxHelperTest.cs

```diff
@@ -90,6 +91,47 @@ public void NameIs_VB()
             toString.NameIs(null).Should().BeFalse();
         }
 
+        [TestMethod]
+        public void IsAssignmentToTuple_IsTrue()
+        {
+            const string code = @"
+public class Sample
+{
+    public void Method()
+    {
+        int a, b;
+        (a, b) = (42, 42);
+    }
+}";
+            var argument = Parse_CS(code).OfType<CSharp.ArgumentSyntax>().First();
+            argument.IsAssignmentToTuple().Should().BeTrue();
```

Comment:
can you also add a test with 
```
int a = 1, b = 2;
var tuple = (a, b);
```
and verify `IsAssignmentToTuple()` returns false?